doc/srcdeps.yaml

```diff
@@ -183,6 +188,62 @@ maven:
     - my-bare-breaker-property
     - my-breaker-property=my-breaker-value
 
+  # `failWithout` - a configuration node to define conditions whose absence will make the outer Maven build fail
```

Comment:
Does it make more sense to have a "negate" `failWith` instead? For instance, `failWith=!srcdeps.enabled` ?